### ISSUE TITLE
Failing to read output files

### ISSUE THREAD

**Issue by nunobernardes99** [DockYard/flame_on#5]
When hitting the flame on button for the first time, it creates the directory `flame_on_output_delete_me` on the project root. After that first time, it doesn't capture anything, it only creates the directory. In the second run, it captures (added the example code to index.html) but at the end of the capture the directory and its contents are deleted and an exception is raised when trying to read the output file to, I assume, display the chart.
```
[error] Process #PID<0.1004.0> raised an exception
** (File.Error) could not read file "flame_on_output_delete_me/1646049518416906-eflambe-output.bggg": no such file or directory
    (elixir 1.13.0) lib/file.ex:355: File.read!/1
    (flame_on 0.2.1) lib/flame_on/capture.ex:30: FlameOn.Capture.watch_output_directory_loop/4
```

Here's a little trace to the console
```
... info about what is capturing ...
Successful finished trace
Output filename: flame_on_output_delete_me/1646049871010006-eflambe-output.bggg
[watch] build started (change: "..")
[debug] Live reload: priv/static/assets/app.js
[watch] build finished
[debug] Live reload: priv/static/assets/app.js
[debug] Live reload: priv/static/assets/app.js
[error] Process #PID<0.1095.0> raised an exception
** (File.Error) could not read file "flame_on_output_delete_me/1646049871010006-eflambe-output.bggg": no such file or directory
    (elixir 1.13.0) lib/file.ex:355: File.read!/1
    (flame_on 0.2.1) lib/flame_on/capture.ex:30: FlameOn.Capture.watch_output_directory_loop/4
[info] GET /dashboard/flame_on
[debug] Processing with Phoenix.LiveView.Plug.page/2
  Parameters: %{"page" => "flame_on"}
  Pipelines: [:admins_only, :browser]
[info] Sent 200 in 30ms

```

**Comment by TheFirstAvenger** [comment]
I just released `0.3.0` which upgrades the underlying eFlambe library to resolve an issue that would account for the file not found issues. Could you try the latest version and let me know if it works any better?

**Comment by TheFirstAvenger** [comment]
Closing this issue for now. If you find that 0.3.0+ doesn't resolve your issue, please reply and let me know.
